src/components/App.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-vars */
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+} from 'react-router-dom';
+import Categories from './categories.js';
+import Books from './books.js';
```

Comment:
- [ ] I couldn't observe a form to add new books, please work on that (don't forget the submit button). The form shouldn't work right now, it needs just to be there. 🌵